pkg/odo/cli/application/application.go

```diff
@@ -2,12 +2,15 @@ package application
 
 import (
 	"fmt"
+	"os"
+	"strings"
+
 	"github.com/pkg/errors"
 	"github.com/redhat-developer/odo/pkg/occlient"
 	"github.com/redhat-developer/odo/pkg/odo/genericclioptions"
-	"github.com/redhat-developer/odo/pkg/odo/util"
-	"os"
-	"strings"
+
+	odoutil "github.com/redhat-developer/odo/pkg/odo/util"
```

Comment:
this is just horrible situation that we got ourselves into :-(

Reply:
I agree haha. I'm going to put the logging stuff into it's own file :+1: